cv/lane_detection/src/line_fitting.py

```diff
@@ -28,13 +28,11 @@ def lane_fitting(points):
     x_width = abs(sorted_points_x[-1][1] - sorted_points_x[0][1])
     y_width = abs(sorted_points_y[-1][0] - sorted_points_y[0][0])
 
-    if (x_width < 15) or (y_width < 15):    # Hard-coded parameter, update maybe
+    if ((x_width > 0.95*y_width) and (x_width < 1.05*y_width) and (x_width < 20 or y_width < 20)) or len(points) < 200:    # Hard-coded parameter, update maybe
         return None
 
-    # print(sorted_points)
-    pts_added = 0
     total_pts = len(points)
-    num_windows = 20
+    num_windows = 30
```

Comment:
Capitalize constants: `NUM_WINDOW = 30` is better since it's more clear that this should be immutable